src/lib/block_if.c

```diff
@@ -372,7 +389,7 @@ blockif_proc(struct blockif_ctxt *bc, struct blockif_elem *be, uint8_t *buf)
```

Comment:
I added the comment, because in a previous version i had `#if 0` and the original version had the bhyve code commented out. Both of which make it blindingly obvious that something from the original source was hacked to make it work on OSX. I'm for removing dead code, but do prefer to leave a breadcrumb indicating that something may have changed due to a hack, just in case someone is debugging this code in a few months time.